Framework/WorkflowAlgorithms/src/ProcessIndirectFitParameters.cpp

```diff
@@ -127,6 +127,12 @@ std::vector<Column_const_sptr> extractColumns(ITableWorkspace const *table,
   return columns;
 }
 
+void populateChiSquaredErrorsWithZeros(std::vector<double> &e,
+                                       std::size_t const &difference) {
+  for (auto i = 0u; i < difference; ++i)
+    e.emplace_back(0.0);
```

Comment:
`std::fill` or `std::fill_n` should do the job as well. I am unsure if you need this entire function just for this purpose.